app/tests/Feature/Http/VotingLists/ShowTest.php

```diff
@@ -193,6 +193,26 @@
     expect($response)->not()->toHaveSelectorWithText('.action-panel', 'Related Votes');
 });
 
+it('only has meta tags for sharing if vote is final', function () {
+    Storage::fake('public');
+    Storage::disk('public')->put('share-pictures/vote-sharepic-1.png', 'test');
+    Storage::disk('public')->put('share-pictures/vote-sharepic-2.png', 'test');
+
+    $response = $this->get('/votes/1');
+    expect($response)->toHaveSelector('[property="twitter:card"]');
+
+    $response = $this->get('/votes/2');
+    expect($response)->not()->toHaveSelector('[property="twitter:card"]');
```

Comment:
Ich würde den Tag-Namen mit in den Selektor aufnehmen, aber nur weil es das etwas besser lesbar macht bzw. dann auf den ersten Blick deutlich wird, dass es um Meta-Tags geht.

```suggestion
    expect($response)->toHaveSelector('meta[property="twitter:card"]');

    $response = $this->get('/votes/2');
    expect($response)->not()->toHaveSelector('meta[property="twitter:card"]');
```